ptero_workflow/api/v1/views.py

```diff
@@ -44,6 +45,26 @@ def get(self, workflow_id):
             LOG.exception('Unexpected exception getting workflow')
             raise
 
+class ExecutionInputsView(Resource):
+    def get(self, execution_id):
+        try:
+            execution_input_data = g.backend.get_execution_inputs(execution_id)
+            return execution_input_data, 200
+        except:
+            LOG.exception('Unexpected exception getting execution inputs')
+            raise
+
+class ExecutionOutputsView(Resource):
+    def put(self, execution_id):
+        try:
+            g.backend.set_execution_outputs(execution_id, request.get_json())
+            return '', 200
+        except OutputsAlreadySet:
+            return '', 403
```

Comment:
I guess if we're going to forbid setting outputs twice, the status code should probably be 409 (Conflict).